src/components/gamesList.js

```diff
@@ -203,6 +203,77 @@ export default function GamesList({ token, games, listType, listTitle }) {
                         Show more
                     </button-showmore>
 
+<<<<<<< HEAD
```

Comment:
This looks like the leftovers from a merge - could you delete the <<<<<<< HEAD and ========= please?